File: src/main/java/net/atos/client/zgw/zrc/model/zaakobjecten/ObjectOverige.java
/*
 * SPDX-FileCopyrightText: 2023 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.zgw.zrc.model.zaakobjecten;

public class ObjectOverige<T> {
    private T overigeData;

    /**
     * Constructor for JSONB deserialization
     */
    public ObjectOverige() {
    }

    /**
     * Constructor with required attributes
     */
    public ObjectOverige(final T overigeData) {
        this.overigeData = overigeData;
    }

    public T getOverigeData() {
        return overigeData;
    }

    public void setOverigeData(final T overigeData) {
        this.overigeData = overigeData;
    }
}

File: src/main/kotlin/nl/info/client/zgw/zrc/model/zaakobjecten/ObjectOverige.kt
/*
 * SPDX-FileCopyrightText: 2023, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.client.zgw.zrc.model.zaakobjecten

import nl.info.zac.util.NoArgConstructor

@NoArgConstructor
data class ObjectOverige<T>(var overigeData: T? = null)
